docs/streaming-kafka-0-10-integration.md

```diff
@@ -141,9 +214,11 @@ For data stores that support transactions, saving offsets in the same transactio
 
 	// begin from the the offsets committed to the database
 	val fromOffsets = selectOffsetsFromYourDatabase.map { resultSet =>
-	  new TopicPartition(resultSet.string("topic")), resultSet.int("partition")) -> resultSet.long("offset")
+	  new TopicPartition(resultSet.string("topic"), resultSet.int("partition")) -> resultSet.long("offset")
 	}.toMap
 
+	import org.apache.spark.streaming.kafka010.ConsumerStrategies.Assign
```

Comment:
This import is in an awkward place, omit it or at least move it to the top of this example.